# edX translation file.
# Copyright (C) 2015 EdX
# This file is distributed under the GNU AFFERO GENERAL PUBLIC LICENSE.
# 
# Translators:
# watcafe, 2014
# kawa <h.kawasaki@nttks.co.jp>, 2014
# h_yoshida <h_yoshida@matsumo-sys.com>, 2014-2015
# sunk <s.kim@nttks.co.jp>, 2014
# Takashi Nagai <ngi644@gmail.com>, 2014
# Shige Usui <ushige@live.com>, 2014
# わたカフェの店長, 2013
msgid ""
msgstr ""
"Project-Id-Version: edx-platform\n"
"Report-Msgid-Bugs-To: openedx-translation@googlegroups.com\n"
"POT-Creation-Date: 2015-03-16 14:10+0000\n"
"PO-Revision-Date: 2015-03-13 02:41+0000\n"
"Last-Translator: h_yoshida <h_yoshida@matsumo-sys.com>\n"
"Language-Team: Japanese (Japan) (http://www.transifex.com/projects/p/edx-platform/language/ja_JP/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: ja_JP\n"
"Plural-Forms: nplurals=1; plural=0;\n"

#: cms/djangoapps/contentstore/course_info_model.py:67
#: cms/djangoapps/contentstore/course_info_model.py:141
msgid "Invalid course update id."
msgstr "無効な講座更新ID"

#: cms/djangoapps/contentstore/course_info_model.py:106
msgid "Course update not found."
msgstr "講座更新がありません。"

#: cms/djangoapps/contentstore/git_export_utils.py:37
msgid ""
"GIT_REPO_EXPORT_DIR not set or path {0} doesn't exist, please create it, or "
"configure a different path with GIT_REPO_EXPORT_DIR"
msgstr "GIT_REPO_EXPORT_DIR が設定されていないか、パス {0} が存在しません。作成するか、GIT_REPO_EXPORT_DIR に違うパスを指定してください。"

#: cms/djangoapps/contentstore/git_export_utils.py:40
msgid ""
"Non writable git url provided. Expecting something like: "
"git@github.com:mitocw/edx4edx_lite.git"
msgstr "書き込みの出来ないgit urlです。「git@github.com:mitocw/edx4edx_lite.git」のような形式が期待値です。"

#: cms/djangoapps/contentstore/git_export_utils.py:42
msgid ""
"If using http urls, you must provide the username and password in the url. "
"Similar to https://user:pass@github.com/user/course."
msgstr "httpのurlを使用している場合、ユーザー名とパスワードをurlに含めてください。「https://user:pass@github.com/user/course」のような形式です。"

#: cms/djangoapps/contentstore/git_export_utils.py:45
msgid "Unable to determine branch, repo in detached HEAD mode"
msgstr "detached HEAD状態のレポジトリではブランチを決定することができません。"

#: cms/djangoapps/contentstore/git_export_utils.py:46
msgid "Unable to update or clone git repository."
msgstr "gitレポジトリでupdateまたはcloneができません。"

#: cms/djangoapps/contentstore/git_export_utils.py:47
msgid "Unable to export course to xml."
msgstr "講座をxmlでエクスポートすることができません。"

#: cms/djangoapps/contentstore/git_export_utils.py:48
msgid "Unable to configure git username and password"
msgstr "gitのユーザー名とパスワードを設定することができません。"

#: cms/djangoapps/contentstore/git_export_utils.py:49
msgid ""
"Unable to commit changes. This is usually because there are no changes to be"
" committed"
msgstr "変更をコミットすることができません。通常コミットする変更が無い場合にこの事象が発生します。"

#: cms/djangoapps/contentstore/git_export_utils.py:51
msgid ""
"Unable to push changes.  This is usually because the remote repository "
"cannot be contacted"
msgstr "変更をプッシュすることができません。通常リモートレポジトリが接続できない場合にこの事象が発生します。"

#: cms/djangoapps/contentstore/git_export_utils.py:53
msgid "Bad course location provided"
msgstr "間違った講座ロケーションです"

#: cms/djangoapps/contentstore/git_export_utils.py:54
msgid "Missing branch on fresh clone"
msgstr "新規クローンにブランチがありません"

#: cms/djangoapps/contentstore/utils.py:32
msgid "My Notes"
msgstr "マイノート"

#: cms/djangoapps/contentstore/management/commands/git_export.py:44
msgid ""
"Take the specified course and attempt to export it to a git repository\n"
". Course directory must already be a git repository. Usage:  git_export <course_loc> <git_url>"
msgstr "gitレポジトリにエクスポートする講座を選んでください。\n講座ディレクトリはgitレポジトリである必要があります。使用方法：git_export <course_loc> <git_url>"

#: cms/djangoapps/contentstore/views/assets.py:245
msgid ""
"File {filename} exceeds maximum size of {size_mb} MB. Please follow the "
"instructions here to upload a file elsewhere and link to it instead: "
"{faq_url}"
msgstr "ファイル{filename}は最大サイズ{size_mb}MBを超えています。 この説明に従ってファイルを別の場所にアップロードしたうえでリンクしてください。 {maxFileSizeRedirectUrl}"

#: cms/djangoapps/contentstore/views/assets.py:296
msgid "Upload completed"
msgstr "アップロード完了"

#: cms/djangoapps/contentstore/views/component.py:245
msgid "HTML"
msgstr "HTML"

#: cms/djangoapps/contentstore/views/component.py:247
msgid "Video"
msgstr "ビデオ"

#: cms/djangoapps/contentstore/views/component.py:266
msgid "Blank"
msgstr "空白"

#: cms/djangoapps/contentstore/views/course.py:330
#: cms/djangoapps/contentstore/views/tests/test_course_index.py:351
msgid "Course has been successfully reindexed."
msgstr "コースの見出し再作成が完了しました。"

#: cms/djangoapps/contentstore/views/course.py:505
msgid "Unscheduled"
msgstr "未定"

#: cms/djangoapps/contentstore/views/course.py:646
msgid ""
"Special characters not allowed in organization, course number, and course "
"run."
msgstr "記号は組織、講座番号、コース開始では使用することができません。"

#: cms/djangoapps/contentstore/views/course.py:662
msgid ""
"There is already a course defined with the same organization and course "
"number. Please change either organization or course number to be unique."
msgstr "既に同様に組織、コース番号で規定されているコースがあります。組織もしくはコース番号のどちらかがユニークとなるように変更をしてください。"

#: cms/djangoapps/contentstore/views/course.py:667
#: cms/djangoapps/contentstore/views/course.py:670
msgid ""
"Please change either the organization or course number so that it is unique."
msgstr "ユニークな情報とするため、組織もしくは講座番号を変更してください。"

#: cms/djangoapps/contentstore/views/course.py:675
msgid ""
"Unable to create course '{name}'.\n"
"\n"
"{err}"
msgstr "講座名「'{name}」の生成に失敗しました。\n{err}"

#: cms/djangoapps/contentstore/views/course.py:905
msgid "Invalid prerequisite course key"
msgstr "無効な必要条件コースキー"

#: cms/djangoapps/contentstore/views/course.py:1374
msgid "invalid JSON"
msgstr "不正なJSONデータです。"

#: cms/djangoapps/contentstore/views/course.py:1383
msgid "must have name of the configuration"
msgstr "この設定の名前を決めてください。"

#: cms/djangoapps/contentstore/views/course.py:1385
msgid "must have at least one group"
msgstr "最低一つのグループが必要です"

#: cms/djangoapps/contentstore/views/course.py:1631
msgid "This group configuration is in use and cannot be deleted."
msgstr "このグループ設定は利用中なので、削除することができません。"

#: cms/djangoapps/contentstore/views/course.py:1645
msgid "This content group is in use and cannot be deleted."
msgstr "このコンテンツグループは利用中なので、削除することができません。"

#: cms/djangoapps/contentstore/views/entrance_exam.py:166
msgid "Entrance Exam - Subsection"
msgstr "入学試験 - 節"

#: cms/djangoapps/contentstore/views/export_git.py:46
msgid "Course successfully exported to git repository"
msgstr "gitレポジトリへの講座エクスポートが完了しました。"

#: cms/djangoapps/contentstore/views/helpers.py:135
msgid "Vertical"
msgstr "縦"

#: cms/djangoapps/contentstore/views/helpers.py:254
msgid "Empty"
msgstr "空"

#: cms/djangoapps/contentstore/views/import_export.py:116
msgid "We only support uploading a .tar.gz file."
msgstr ".tar.gz 形式のファイルのアップロードのみがサポートされています。"

#: cms/djangoapps/contentstore/views/import_export.py:154
msgid "File upload corrupted. Please try again"
msgstr "ファイルアップロードが失敗しました。再度、お試しください。"

#: cms/djangoapps/contentstore/views/import_export.py:251
msgid "Could not find the {0} file in the package."
msgstr "パッケージ内にファイル {0} が見当たりませんでした。"

#: cms/djangoapps/contentstore/views/item.py:473
msgid "Invalid data"
msgstr "不正なデータ"

#: cms/djangoapps/contentstore/views/item.py:475
msgid "Invalid data ({details})"
msgstr "不正なデータ ({details})"

#: cms/djangoapps/contentstore/views/item.py:578
msgid "Duplicate of {0}"
msgstr "'{0}' の重複"

#: cms/djangoapps/contentstore/views/item.py:580
msgid "Duplicate of '{0}'"
msgstr "'{0}' の重複"

#: cms/djangoapps/contentstore/views/item.py:794
#, python-format
msgid "Students must score {score}% or higher to access course materials."
msgstr "受講生は学習教材にアクセスするには {score}% o以上得点しなければなりません。"

#: cms/djangoapps/contentstore/views/item.py:1028
msgid "{section_or_subsection} \"{display_name}\""
msgstr "{section_or_subsection} \"{display_name}\""

#: cms/djangoapps/contentstore/views/library.py:140
msgid "Unable to create library - missing required field '{field}'"
msgstr "ライブラリを作成できませんでした。必要なフィールド'{field}'がありません。"

#: cms/djangoapps/contentstore/views/library.py:145
msgid ""
"Unable to create library '{name}'.\n"
"\n"
"{err}"
msgstr "ライブラリ'{name}'を作成できません。\n\n{err}"

#: cms/djangoapps/contentstore/views/library.py:151
msgid ""
"There is already a library defined with the same organization and library "
"code. Please change your library code so that it is unique within your "
"organization."
msgstr "既に同様に組織、ライブラリコードで規定されているコースがあります。組織もしくはライブラリコードのどちらかがユニークとなるように変更をしてください。"

#: cms/djangoapps/contentstore/views/transcripts_ajax.py:437
msgid "Incoming video data is empty."
msgstr "入力されたビデオデータは空です。"

#: cms/djangoapps/contentstore/views/transcripts_ajax.py:442
msgid "Can't find item by locator."
msgstr "ロケーターでアイテムを見つけることができません。"

#: cms/djangoapps/contentstore/views/transcripts_ajax.py:445
msgid "Transcripts are supported only for \"video\" modules."
msgstr "字幕は「ビデオ」モジュールのみ対応します。"

#: cms/djangoapps/contentstore/views/user.py:93
msgid "Insufficient permissions"
msgstr "権限が不十分です"

#: cms/djangoapps/contentstore/views/user.py:105
msgid "Could not find user by email address '{email}'."
msgstr "電子メールアドレス「{email}」でユーザを見つけることができませんでした．"

#: cms/djangoapps/contentstore/views/user.py:138
msgid "User {email} has registered but has not yet activated his/her account."
msgstr "ユーザ {email} は登録済みですが、まだアクティベーションが完了していません。"

#: cms/djangoapps/contentstore/views/user.py:151
msgid "No `role` specified."
msgstr "「ロール」が指定されていません。"

#: cms/djangoapps/contentstore/views/user.py:170
msgid "Invalid `role` specified."
msgstr "無効な「ロール」が指定されています。"

#: cms/djangoapps/contentstore/views/user.py:174
msgid "You may not remove the last Admin. Add another Admin first."
msgstr "最後の管理者を削除することはできません。先に別の管理者を追加してください。"

#. Translators: This is the status of an active video upload
#: cms/djangoapps/contentstore/views/videos.py:45
msgid "Uploading"
msgstr "アップデート中"

#. Translators: This is the status for a video that the servers are currently
#. processing
#: cms/djangoapps/contentstore/views/videos.py:47
msgid "In Progress"
msgstr "実施中"

#. Translators: This is the status for a video that the servers have
#. successfully processed
#: cms/djangoapps/contentstore/views/videos.py:49
msgid "Ready"
msgstr "準備完了"

#. Translators: This is the status for a video that the servers have failed to
#. process
#: cms/djangoapps/contentstore/views/videos.py:51
msgid "Failed"
msgstr "失敗"

#. Translators: This is the status for a video for which an invalid
#. processing token was provided in the course settings
#: cms/djangoapps/contentstore/views/videos.py:54
msgid "Invalid Token"
msgstr "無効なトークン"

#. Translators: This is the status for a video that is in an unknown state
#: cms/djangoapps/contentstore/views/videos.py:56
msgid "Unknown"
msgstr "不明"

#. Translators: This is the header for a CSV file column
#. containing URLs for video encodings for the named profile
#. (e.g. desktop, mobile high quality, mobile low quality)
#: cms/djangoapps/contentstore/views/videos.py:127
msgid "{profile_name} URL"
msgstr "{profile_name} URL"

#: cms/djangoapps/contentstore/views/videos.py:133
msgid "Duration"
msgstr "期間"

#: cms/djangoapps/contentstore/views/videos.py:134
msgid "Date Added"
msgstr "追加日"

#: cms/djangoapps/contentstore/views/videos.py:135
msgid "Video ID"
msgstr "動画ID"

#. Translators: This is listed as the duration for a video that has not
#. yet reached the point in its processing by the servers where its
#. duration is determined.
#: cms/djangoapps/contentstore/views/videos.py:149
msgid "Pending"
msgstr "ペンディング"

#. Translators: This is the suggested filename when downloading the URL
#. listing for videos uploaded through Studio
#: cms/djangoapps/contentstore/views/videos.py:172
msgid "{course}_video_urls"
msgstr "{course}動画URL"

#: cms/djangoapps/course_creators/models.py:33
msgid "unrequested"
msgstr "要求されていない"

#: cms/djangoapps/course_creators/models.py:34
msgid "pending"
msgstr "ペンディング"

#: cms/djangoapps/course_creators/models.py:35
msgid "granted"
msgstr "許可済み"

#: cms/djangoapps/course_creators/models.py:36
msgid "denied"
msgstr "不許可"

#: cms/djangoapps/course_creators/models.py:39
msgid "Studio user"
msgstr "Studio ユーザ"

#: cms/djangoapps/course_creators/models.py:41
msgid "The date when state was last updated"
msgstr "状態が最後に更新された日時"

#: cms/djangoapps/course_creators/models.py:43
msgid "Current course creator state"
msgstr "現在の講座制作状況"

#: cms/djangoapps/course_creators/models.py:44
msgid ""
"Optional notes about this user (for example, why course creation access was "
"denied)"
msgstr "このユーザへの補足説明 (たとえば、講座制作画面へのアクセス拒否理由、等)"

#: cms/djangoapps/models/settings/course_metadata.py:128
msgid "Incorrect format for field '{name}'. {detailed_message}"
msgstr "次のフィールドがフォーマットが不正です：'{name}'. {detailed_message}"
